### ISSUE TITLE
Invalid repository in the CIC OCP UPI setup

### ISSUE THREAD

**Issue by vmorris** [IBM/z_ansible_collections_samples#68]
Attempting to setup the necessary repositories for installation and:

`Error: 'openstack-16-tools-for-rhel-8-s390x-rpms' does not match a valid repository ID.`

```
[root@cic-manager ocp_upi]# subscription-manager repos \
>   --enable=rhel-8-for-s390x-baseos-rpms \
>   --enable=openstack-16-tools-for-rhel-8-s390x-rpms \
>   --enable=ansible-2.8-for-rhel-8-s390x-rpms \
>   --enable=rhel-8-for-s390x-appstream-rpms
Error: 'openstack-16-tools-for-rhel-8-s390x-rpms' does not match a valid repository ID. Use "subscription-manager repos --list" to see valid repositories.
Repository 'rhel-8-for-s390x-baseos-rpms' is enabled for this system.
Repository 'ansible-2.8-for-rhel-8-s390x-rpms' is enabled for this system.
Repository 'rhel-8-for-s390x-appstream-rpms' is enabled for this system.
```

**Comment by zhengxiaomei123** [comment]
Hi @vmorris , Thanks for your feedback. After our verification, the repo is not required on s390x.  We will update the ReadMe later to make it clear.

**Comment by zhengxiaomei123** [comment]
close this issue, since have merged the PR #71 . @vmorris please try the latest ReadMe. Thanks.
